design/design_resort.md

```diff
@@ -0,0 +1,62 @@
+# Resort Write-Up
+
+This document features the writeup for the levels to be included in resort chapter of newleste.p8
+
+# Levels
+
+## Summary
+
+- The chapter starts with introductory levels that introduce to main mechanics.
+- Then the mess levels tests those mechanics.
+- The roof levels test the skill of player in those mechanics along with a boss fight.
```

Comment:
The roof levels combine the previously introduced mechanics along with the added challenge of the Oshiro boss fight.